build.gradle

```diff
@@ -8,6 +9,7 @@ buildscript {
     classpath 'com.android.tools.build:gradle:3.0.1'
     classpath 'com.netflix.nebula:gradle-lint-plugin:7.4.0'
     classpath 'com.nabilhachicha:android-native-dependencies:0.1.2'
+    classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
```

Comment:
Kotlin could provide a lot of neat language features and just a general pleasure to programming for our app. I'm glad to see you taking my lead and adopting it for your work.

Reply:
GitHub is no place for joking around and telling lies, @gioneill!